ui/src/Pages/VideoPlayer/Controls/Actions/Subtitles.jsx

```diff
@@ -12,9 +12,13 @@ function VideoActionSubtitles() {
     video: store.video,
   }));
 
-  const toggleSubtitles = useCallback(() => {
-    dispatch(toggleShowSubSwitcher());
-  }, [dispatch]);
+  const toggleSubtitles = useCallback(
+    (e) => {
+      e.target.blur();
```

Comment:
I think it'd be cleaner if we create a custom component called UnfocusableButton which takes in a callback and calls the callback then e.target.blur. this would remove a lot of duplicate code